lib/recurly.js

```diff
@@ -66,6 +71,7 @@ export class Recurly extends Emitter {
       token: bind(this, bankAccount.token),
       bankInfo: bind(this, bankAccount.bankInfo)
     };
+    this.fraud = new Fraud(this);
```

Comment:
I'm thinking this should just run in the parent context, otherwise it will execute within the hosted fields individually as well.

I'm thinking a refactor is going to help with the separation of concerns between parent and field contexts quite a bit. That should follow this.